lib/init.coffee

```diff
@@ -59,34 +60,31 @@ module.exports =
       lintOnFly: true
 
       lint: (textEditor) =>
+        rules = []
         filePath = textEditor.getPath()
         fileText = textEditor.getText()
-        projectConfigPath = @getConfig(filePath)
+        projectConfig = @getConfig(filePath)
 
         if !fileText
           return Promise.resolve([])
 
-        parameters = [filePath]
-
-        if !projectConfigPath || !projectConfigPath.configPath
-          if !@onlyRunWhenConfig
+        if !projectConfig || !projectConfig.configPath
+          if @onlyRunWhenConfig
             atom.notifications.addError 'Pug-lint config not found'
-          return Promise.resolve([])
+            return Promise.resolve([])
```

Comment:
That's the point, I want `pug-lint` to run on all my .pug files so it shows syntax errors as well. With the current plugin my files won't get linted at all if I don't have a config in every project I use PUG in.